src/types.ts

```diff
@@ -8,7 +8,8 @@ export interface AppSettings {
   autoSyncSaves: boolean
   customWinePaths: Array<string>
   darkTrayIcon: boolean
-  defaultInstallPath: string
+  defaultInstallPath: string,
+  discordrpc: boolean,
```

Comment:
```suggestion
  discordRPC: boolean,
```

Maybe? Would need to changed everywhere.

Reply:
Because RPC are an acronym, they should be in upper case.